packages/js/test/plugins/auctionHouseModule/executePartialSale.test.ts

```diff
@@ -61,12 +61,20 @@ test('[auctionHouseModule] execute partial sale on an Auction House', async (t:
     .run();
 
   // Then the user must receive 3 Tokens.
-  const buyerToken = await mx
+  const buyerTokens = await mx
     .nfts()
     .findByToken({ token: purchase.asset.token.address })
     .run();
 
-  t.equal(buyerToken.token.amount.basisPoints.toNumber(), 3);
+  t.equal(buyerTokens.token.amount.basisPoints.toNumber(), 3);
+
+  // And then the seller must have 2 Tokens left.
+  const sellerTokens = await mx
+    .nfts()
+    .findByToken({ token: listing.asset.token.address })
+    .run();
+
+  t.equal(sellerTokens.token.amount.basisPoints.toNumber(), 2);
```

Comment:
Has mentioned on the other thread let's also check the `delegatedAmount` here. And let's give more token to the seller initially to make sure we only offer the amount listed.